### DISCUSSION TITLE
ScriptReference/Application-isBatchMode

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#218] [Notes]
# ScriptReference/Application-isBatchMode

https://docs.unity3d.com/ScriptReference/Application-isBatchMode.html

**Comment by mvi** [comment]
Prior to Unity 2018.2 you can use [`InternalEditorUtility.inBatchMode`](https://sabresaurus.com/unity-api-versioner/?api=InternalEditorUtility.inBatchMode).
